app/views/thredded/private_topic_mailer/message_notification.html.erb

```diff
@@ -1,7 +1,5 @@
-<blockquote><%= @topic.posts.first.filtered_content(self) %></blockquote>
-<%                           # as this is only sent on first post, doesn't need to define order %>
+<blockquote><%= @topic.posts.last.content %></blockquote>
```

Comment:
The post should just be passed directly from the place that creates this notification, i.e.:
https://github.com/thredded/thredded/blob/019281c256f0c8b23e737d58863af90a04d16260/app/notifiers/thredded/email_notifier.rb#L21